dmscripts/helpers/env_helpers.py

```diff
@@ -1,9 +1,9 @@
 def get_api_endpoint_from_stage(stage, app='api'):
-    """Return the full URL of given API or Search API environment.
+    """Return the full URL of given API, Search API or web URL environment.
 
     :param stage: environment name. Can be one of 'preview', 'staging',
                   'production' or 'dev' (aliases: 'local', 'development').
-    :param app: should be either 'api' or 'search-api'
+    :param app: should be either 'api', 'search-api' or 'www'
```

Comment:
I think the idea was that the email could be sent on any environment and still link to the right domain (e.g. during QA on staging). However there are other email templates that use hardcoded URLs, so this may not be a huge problem. Or do you mean determining the URL with something like `{% if stage == "preview" %}` in the template itself?